### PR TITLE
docs: add section elaborating on signed transactions for chch/watcher

### PR DESCRIPTION
**Pull request by pdobacz**
Following a request from @achiurizo, this adds a section elaborating handling of signed transactions in https://github.com/omisego/elixir-omg/ to the intregration docs transaction section.

I tried to follow conventions and blend in with the rest of the doc, but not sure if I got this right. Same goes for level of detail.

Also, there's an alternative location for this documentation: it could be put wherever the handling of such encoded signed transactions is done (`elixir-omg` currently). So if this doesn't fit in here, let's discuss.

### CODE REVIEW

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 1]
```diff
@@ -90,13 +90,36 @@ outputType ::= uint256
...
+Where
+```
+signatures ::= bytes
+rawTransaction ::= transaction
+```
+
+and are transferred and stored RLP-encoded.
+
+Child chain and watcher both expect, for a signed transaction to be valid, that:
+ - every input corresponds to a signature by the respective output's owner (`outputGuard`)
```
Comment (kevsul) [inline plasma_framework/docs/integration-docs/integration-doc.md]: I think this makes more sense (but feel free to keep the original)
```suggestion
 - every input has a corresponding signature by the respective output's owner (`outputGuard`)
```

File: plasma_framework/docs/integration-docs/integration-doc.md [hunk 2]
```diff
@@ -90,13 +90,36 @@ outputType ::= uint256
...
+
+Signed transactions are:
+
+```
+signedTransaction ::= [signature] rawTransaction
+```
+
+Where
+```
+signatures ::= bytes
```
Comment (kevsul) [inline plasma_framework/docs/integration-docs/integration-doc.md]:
```suggestion
signature ::= bytes
```

**Comment by pdobacz** [comment]
hm, why would the Python tests fail here?

**Comment by pgebal** [comment]
> hm, why would the Python tests fail here?

I have the same issue, these are gas cost tests. I don't know how to solve that yet.

**Comment by boolafish** [comment]
> hm, why would the Python tests fail here?

Sigh, the latest release of `ganache-cli v6.8.1` breaks this: https://github.com/trufflesuite/ganache-cli/releases/tag/v6.8.1

**Pull request merged**